src/commands/targeting/get.test.ts

```diff
@@ -0,0 +1,127 @@
+    const authFlags = [
+        '--project', projectKey,
+        '--client-id', 'test-client-id',
+        '--client-secret', 'test-client-secret'
+    ]
+
+    const mockTargeting = [
+        {
+            '_feature': '642d9de38bbfc09ad0657e05',
+            '_environment': '637cfe8195279288bc08cb61',
+            '_createdBy': 'google-oauth2|112946554154333301873',
+            'status': 'active',
+            'startedAt': '2023-04-05T16:12:19.911Z',
+            'updatedAt': '2023-04-05T16:12:19.922Z',
+            'targets': [
+                {
+                    '_id': '642d9de38bbfc09ad0657e1c',
+                    'audience': {
+                        'name': 'All Users',
+                        'filters': {
+                            'operator': 'and',
+                            'filters': [{ 'type': 'all' }]
+                        }
+                    },
+                    'distribution': [{
+                        '_variation': '642d9de38bbfc09ad0657e09',
+                        'percentage': 1
+                    }]
+                }
+            ],
+            'readonly': false
+        },
+        {
+            '_feature': '642d9de38bbfc09ad0657e05',
+            '_environment': '637cfe8195279288bc08cb62',
+            '_createdBy': 'google-oauth2|112946554154333301873',
+            'status': 'inactive',
+            'updatedAt': '2023-04-05T16:12:19.923Z',
+            'targets': [{
+                '_id': '642d9de38a131fadcb5051b0',
+                'audience': {
+                    'name': 'All Users',
+                    'filters': {
+                        'operator': 'and',
+                        'filters': [{ 'type': 'all' }]
+                    }
+                },
+                'distribution': [{
+                    '_variation': '642d9de38bbfc09ad0657e09',
+                    'percentage': 1
+                }]
+            }],
+            'readonly': false
+        },
+        {
+            '_feature': '642d9de38bbfc09ad0657e05',
+            '_environment': '637cfe8195279288bc08cb63',
+            '_createdBy': 'google-oauth2|112946554154333301873',
+            'status': 'inactive',
+            'updatedAt': '2023-04-05T16:12:20.174Z',
+            'targets': [],
+            'readonly': false
+        },
+        {
+            '_feature': '642d9de38bbfc09ad0657e05',
+            '_environment': '647f62ae749bbe90fb222070',
+            'status': 'inactive',
+            'targets': [],
+            'readonly': false
+        }
+    ]
+
+    dvcTest()
+        .nock(BASE_URL, (api) => api
+            .get(`/v1/projects/${projectKey}/features/${featureKey}/configurations`)
+            .reply(200, mockTargeting)
+        )
+        .stdout()
+        .command(['targeting get', featureKey, ...authFlags])
+        .it('returns all targeting for a feature',
+            (ctx) => {
+                expect(ctx.stdout).to.contain(JSON.stringify(mockTargeting, null, 2))
```

Comment:
Do you have thoughts on JSON parsing the ctx.stdout rather than stringifying the expected value? It has the advantage of not needing to worry about the spacing, but there might be downsides I havent considered

Reply:
I had considered doing it that way because it would make this `expect` more readable, but chose to go with this approach so that it _does_ validate the whitespace — meaning that we expect non-headless commands to output formatted json, so this would fail if that's not the case.

That being said, my next PR will change the targeting output to an [oclif tree](https://github.com/oclif/core/tree/a2efc040855969678a6aaf14d3ceb1d25d7a8411/src/cli-ux#uxtree) rather than JSON, so I'll likely need to switch this to some sort of snapshot anyway

Reply:
Testing for the whitespace feels more like were testing the `writer.showResults` function over and over, but if its going to be a moot point soon anyway then its probably not even worth thinking about